BetterShardsBukkit/src/main/java/vg/civcraft/mc/bettershards/listeners/BetterShardsListener.java

```diff
@@ -74,7 +72,7 @@ public BetterShardsListener(){
 		plugin = BetterShardsPlugin.getInstance();
 		db = BetterShardsPlugin.getDatabaseManager();
 		pm = BetterShardsPlugin.getPortalManager();
-		config = plugin.GetConfig();
+		config = plugin.getConfig();
```

Comment:
Without the savedefault and other boilerplate, the default config.yml won't matter.